packages/xo-lib/README.md

```diff
@@ -118,8 +118,8 @@ xo.on('closed', () => {
 ```
 
 ```javascript
-xo.on('notification', function (notif) {
```

Comment:
I understand someone is trying to avoid writing out `notification`, I don't think that's valuable. You could write `notify`, it'd be better than inventing a token.

Here I'm selecting `notice` instead as a `notice` is the thing that represents a `notification`.